pkg/system/xattrs_linux.go

```diff
@@ -1,35 +1,21 @@
 package system
 
-import (
-	"syscall"
-	"unsafe"
-)
+import "golang.org/x/sys/unix"
 
 // Lgetxattr retrieves the value of the extended attribute identified by attr
 // and associated with the given path in the file system.
 // It will returns a nil slice and nil error if the xattr is not set.
 func Lgetxattr(path string, attr string) ([]byte, error) {
```

Comment:
@AkihiroSuda I guess you meant to remove the Lsetxattr wrapper function? The Lgetxattr wrapper has additional functionality (resizing the buffer if the xattrs don't fit) which is not covered by the version from x/sys/unix.